### ISSUE TITLE
Add a role to a user

### ISSUE THREAD

**Issue by Zorin95670** [ditrit/leto-modelizer-admin#35]
## Description

blocked by #84 

On user page, add button to add a role(s) on the user.

On button click, open pop to chose one or many roles to add on user.

Update Rolestable to make optional multiselection.

![image](https://github.com/ditrit/leto-modelizer-admin/assets/6179956/e22d8868-46d4-4c80-b466-b1625315cdcf)
![image](https://github.com/ditrit/leto-modelizer-admin/assets/6179956/0eeb44b0-5b05-4389-a40b-8c24c552b608)

**Comment by justinehell** [comment]
Task put on hold, will be done when the new api is up.
